src/app/pages/landing/landingPage.component.ts

```diff
@@ -38,13 +38,13 @@ export class LandingPageComponent implements OnInit {
     return this.vendorLoginPressed = true;
   }
 
-  customerLogin() {
-     return this.backend.customerLogin(this.loginForm.username, this.loginForm.password)
-  }
+  // customerLogin() {
```

Comment:
You probably want to undo this change here too. Just uncomment the parts you commented out.